.gitignore

```diff
@@ -1 +1,129 @@
-TODO
```

Comment:
Interesting. I usually just agree with whatever .gitignore GitHub creates (which is what I've added). In this way if people don't have a global one, they won't accidentally commit some wrong stuff. No problem though, I will remove. Should we at least keep ".tox" and "node_modules" there?